recipe/bld.bat

```diff
@@ -1,18 +1,5 @@
-copy %RECIPE_DIR%\build.sh .
-
-FOR /F "delims=" %%i IN ('cygpath.exe -u "%LIBRARY_PREFIX%"') DO set "LIBRARY_PREFIX=%%i"
-FOR /F "delims=" %%i IN ('cygpath.exe -u "%PREFIX%"') DO set "PREFIX=%%i"
-FOR /F "delims=" %%i IN ('cygpath.exe -u "%PYTHON%"') DO set "PYTHON=%%i"
-FOR /F "delims=" %%i IN ('cygpath.exe -u "%RECIPE_DIR%"') DO set "RECIPE_DIR=%%i"
-FOR /F "delims=" %%i IN ('cygpath.exe -u "%SP_DIR%"') DO set "SP_DIR=%%i"
-FOR /F "delims=" %%i IN ('cygpath.exe -u "%SRC_DIR%"') DO set "SRC_DIR=%%i"
-FOR /F "delims=" %%i IN ('cygpath.exe -u "%STDLIB_DIR%"') DO set "STDLIB_DIR=%%i"
-
-bash build.sh
-
+xcopy /y "%SRC_DIR%"\cacert.pem "%LIBRARY_PREFIX%"\ssl\
```

Comment:
more correct quoting would be 
```xcopy /y "%SRC_DIR%\cacert.pem" "%LIBRARY_PREFIX%\ssl\"```

but that's nitpicking.  What you have here should work.